### ISSUE TITLE
knowledge distillation train

### ISSUE THREAD

**Issue by WiYj** [OpenRLHF/OpenRLHF#1146]
Can knowledge distillation training support the selection of top k logits distillation and multi-machine and multi-card training?
